### DISCUSSION TITLE
The image is not fully filled in the container

### DISCUSSION THREAD

**Discussion by shuwenjunn** [fabricjs/fabric.js#8285] [Q&A] [answered]
## Version
5.0.0
## Test Case

- [ ] [Browser Issue Template](https://jsfiddle.net/efoq0jxk/19/)

## Information about environment

browser
Which browsers?
chrome

## Steps To Reproduce
1. Open https://jsfiddle.net/efoq0jxk/19/
2. Select the picture
Description:
The width of the original image is 200px，when I set the width of the container to 400px，I want the image to fill the container completely，is there any way？
<img width="620" alt="image" src="https://user-images.githubusercontent.com/33014124/189797269-e80d9912-4c5a-41e5-a900-65960a0ab6d9.png">

**Comment by ShaMan123** [comment]
It seems you are looking for a `resizeMode` feature, https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit
Actually sounds like a decent feature. We are deep in TS migration and have a lot of things on our list so don't wait (and is low priority).
That being said, the common way to do it is using scaling. Use `findScaleToFit`,`findScaleToCover`

**Comment by chloe-bm** [comment, marked as answer]
This should work:

```ts
    fabric.Image.fromURL('https://cdn.chime.me/image/fs/user-info/2021115/11/w200_original_d6f62867-9576-4820-b1fa-754f2595d493.png', function (oImg) {
      canvas.add(oImg);
      oImg.scale(400 / oImg._element.width)
    });
```

```ts
    fabric.Image.fromURL('https://cdn.chime.me/image/fs/user-info/2021115/11/w200_original_d6f62867-9576-4820-b1fa-754f2595d493.png', function (oImg) {
      canvas.add(oImg);
      oImg.scaleToWidth(400)
    });
```

**Reply by ShaMan123** [reply]
forgot about those
good advice
